app/index.html

```diff
@@ -0,0 +1,91 @@
+<html>
+<head>
+    <title>Bridge Researcher Portal</title>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/1.12.0/semantic.min.css"/>
+</head>
+<body class="HolyGrail">
+    <header>
+        <h2>Bridge Researcher Portal</h2>
+        <a href="#signOut">Sign Out</a>
+    </header>
+    <div class="HolyGrail-body">
```

Comment:
Rename HolyGrail layout. Also, it's not flexible, maybe switch to menu in semantic ui.